web/src/main/webapp/index.html

```diff
@@ -1,47 +1,54 @@
+<!--
+Copyright 2016 The Johns Hopkins University Applied Physics Laboratory LLC
+All rights reserved.
+ 
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+
 <html>
    <head>
+      <link rel="stylesheet" href="resources/css/custom.css">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
       <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-      <title> Dorset Frontend </title>
+      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"/></script>
+      <script src="resources/js/custom.js"></script>
+      <title> Dorset Web Demo </title>
    </head>
-   <body>
-      <h2>
-         Welcome to the Dorset Front End Demo!
-      </h2>
-      <form>
-         Enter Question:<br>
-         <input type="text" name="question" id="questionInputId">          
-         <button type="button" onclick="submitQuestion()">Submit</button><br>
-         Answer: <br>
-         <input type="text" name="answer" id="answerOutputId">
-      </form>
+   <body id="background">
+         <div class="container">
+         	<br><br><br><br><br><br>
```

Comment:
Spacing should be handled by adding margin or padding to elements rather than adding line breaks. I'd add a custom class and include that in your css file.